client/js/components/user/CustomerSettings/CustomerSettingsBranding.vue

```diff
@@ -158,8 +161,12 @@ export default {
         this.updateBranding(payload)
         this.saveBranding(this.brandingId).then(() => {
           dplan.notify.notify('confirm', Translator.trans('confirm.saved'))
-          this.isLogoDeleted = false
           this.isBusy = false
+
+          if (this.isLogoDeleted) {
+            this.uploadedFileId = ''
+            this.isLogoDeleted = false
+          }
```

Comment:
do we need the if clause here?

Reply:
its jut for to set the isLogoDeleted to false only if its true, but it works without the check, too. I thought, it would be more correct this way

Reply:
otherwise it has the default false

Reply:
sorry its more important here for if islogodeleted to set uploadedFileId to empty string. this is important here

Reply:
okay, I see. I will change it here again. thanks for the advice

Reply:
done here https://github.com/demos-europe/demosplan-core/pull/2255/commits/698ad984a17cc306de6a1c3abc7d7b9ebac542ec